src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The universal selector '*' is used here which impacts performance. It's better to set styles only for elements that require them. Consider removing the universal selector and applying the reset styles to specific elements or using a more efficient selector.